source/extensions/filters/http/jwt_authn/authenticator.cc

```diff
@@ -272,6 +275,42 @@ void AuthenticatorImpl::verifyKey() {
   handleGoodJwt(/*cache_hit=*/false);
 }
 
+void AuthenticatorImpl::addJWTClaimToHeader(const std::string& claim_name,
+                                            const std::string& header_name) {
+  StructUtils payload_getter(jwt_->payload_pb_);
+  const ::Envoy::ProtobufWkt::Value* claim_value;
+  const auto status = payload_getter.GetValue(claim_name, claim_value);
+  if (status == StructUtils::OK) {
+    if (claim_value->kind_case() == Envoy::ProtobufWkt::Value::kStringValue) {
+      headers_->addCopy(Http::LowerCaseString(header_name), claim_value->string_value());
+      ENVOY_LOG(debug, "--------claim : {} with value : {} is added to the header : {} -----------",
+                claim_name, claim_value->string_value(), header_name);
+    } else if (claim_value->kind_case() == Envoy::ProtobufWkt::Value::kNumberValue) {
+      if (claim_value->number_value() < 0 ||
+          claim_value->number_value() >=
+              static_cast<double>(std::numeric_limits<uint64_t>::max())) {
+        ENVOY_LOG(debug, "--------claim : {} is not a valid integer -----------", claim_name);
+      }
+      const uint64_t int_claim_value = static_cast<uint64_t>(claim_value->number_value());
+      headers_->addCopy(Http::LowerCaseString(header_name), std::to_string(int_claim_value));
+      ENVOY_LOG(debug, "--------claim : {} with value : {} is added to the header : {} -----------",
+                claim_name, std::to_string(int_claim_value), header_name);
+    } else if (claim_value->kind_case() == Envoy::ProtobufWkt::Value::kBoolValue) {
+      const std::string str_bool_claim = claim_value->bool_value() ? "true" : "false";
+      headers_->addCopy(Http::LowerCaseString(header_name), str_bool_claim);
+      ENVOY_LOG(debug, "--------claim : {} with value : {} is added to the header : {} -----------",
+                claim_name, str_bool_claim, header_name);
+    } else {
+      ENVOY_LOG(debug, "--------claim : {} is not of type string, int or bool -----------",
```

Comment:
"is not a primitive types of int, string or bool.'

Reply:
It seems that following two lines are duplicated 3 time
```
headers->addCopy
ENVOY_LOG(debug...
```

Try to remove the duplication codes.